api/server.go

```diff
@@ -37,6 +37,35 @@ type ExtraOptions struct {
 
 	// ReadTimeout is the maximum duration for reading the entire request, including the body.
 	ReadTimeout time.Duration
+
+	// MaxAccountsAPIResults is the maximum number of combined AppParams, AppLocalState, AssetParams,
+	// and AssetHolding resources per address that can be returned by the /v2/accounts endpoints.
+	// If an address exceeds this number, a 400 error is returned. Zero means unlimited.
+	MaxAccountsAPIResults uint64
```

Comment:
This is the same name as used by algod — I had asked @barnjamin if the word "creatables" was widely used and it is not. Resources is ambiguous but kind of OK by me. Agreed "Max Accounts API Results" is confusing especially since you have other various configurable limits in the indexer now. Maybe `MaxNestedAccountResources` ...

Reply:
I agree with you. API results just seems like an alias for limit, but I'm not a huge fan of my suggestions either. Maybe MaxAccountNestedObjects?

Reply:
That seems good.. will switch